sci-misc/boinc/boinc-7.8.4.ebuild

```diff
@@ -147,7 +147,7 @@ pkg_preinst() {
 }
 
 pkg_postinst() {
-	gnome2_icon_cache_update
+	use X && gnome2_icon_cache_update
```

Comment:
As already mentioned in #6530, please don't do this conditional to use X. You would have to be sure there are no icons installed then on each bump, etc. Much better to just do what gnome2.eclass does in this case. Put back the savelist in preinst, and then actually use the result in the same way gnome2.eclass does.